src/query/api/v1/handler/prometheus/common.go

```diff
@@ -361,3 +361,22 @@ func RenderSeriesMatchResultsJSON(
 
 	return jw.Close()
 }
+
```

Comment:
Add some docs (either to the README, or linked from there) including sample usage?

Reply:
I'll add docs in follow up pr (have that on my to-do)